errbot/backends/telegram_messenger.py

```diff
@@ -256,6 +258,8 @@ def _handle_message(self, message):
             log.warning("Unhandled message type (not a text message) ignored")
             return
 
+        if re.match('\/.[^ ]*@{}'.format(self.botname), message.text):
+            message.text = message.text.strip("@{}".format(self.botname))
```

Comment:
I don't see how this could ever work. Strip removes characters from beginning/end of string, it can't remove a substring in the middle of a string.